cmd/src/server.rs

```diff
@@ -473,6 +474,18 @@ fn check_system_config(config: &TiKvConfig) {
             "err" => %e
         );
     }
+    // Check blob file dir is empty when titan is disabled
+    if !config.rocksdb.titan.enabled {
+        let titan_dir = if config.rocksdb.titan.dirname.is_empty() {
+            "titandb"
+        } else {
+            config.rocksdb.titan.dirname.as_str()
+        };
+        if let Err(e) = tikv_util::config::check_data_dir_empty(titan_dir, "blob") {
+            set_panic_mark();
```

Comment:
Usually, `panic_mark` indicates something wrong with the data. Here is just a wrong config.

Reply:
That sure is one way to interpret it. I will remove it since no **irreversible** corruption has occurred.